doc/tagbar.txt

```diff
@@ -437,60 +437,60 @@ KEY MAPPINGS                                                     *tagbar-keys*
 The following mappings are valid in the Tagbar window:
 
 <F1>/?        Display key mapping help.
-                Map option: tagbar_map_help
+                Map option: *tagbar_map_help*
```

Comment:
```suggestion
                Map option: |tagbar_map_help|
```
As per the `:help help-writing` documentation within vim, the `*tag*` syntax is used to define a tag, and the `|link|` is used to link to a tag. In this case we want to use the link to a tag option so please change all these to use the `|` characters instead.

Reply:
Okay, sure I'll run `%s/\*\(tagbar_map_\w\+\)\*/|\1|/g`.
It should be good now. This is my first time working on a help file btw.